source/manage-apps/configure/config/auth.txt

```diff
@@ -201,3 +202,7 @@ collection for your app in ``/auth/custom_user_data.json``.
      - The name of the field in custom user data documents that
        contains the user ID of the application user the document
        describes.
+
+   * - | ``on_user_creation_function_name``
+       | String
+     - The name of the :ref:`user creation function <user-creation-function>`.
```

Comment:
```suggestion
     - The name of the :ref:`user creation Atlas Function <user-creation-function>`.
```

Reply:
gonna leave these - "user creation function" doesn't need to reference the service name in the same way that a "trigger function" isn't a "Trigger Atlas Function"